native/src/routePlannerFrontEnd.cpp

```diff
@@ -589,6 +589,7 @@ bool RoutePlannerFrontEnd::findGpxRouteSegment(SHARED_PTR<GpxRouteApproximation>
 		gctx->routeCalculations++;
 		RoutingContext* cp = new RoutingContext(gctx->ctx);
 		res = searchRouteInternalPrepare(cp, start->pnt, target->pnt, nullptr);
+		prepareResult(cp, res);
```

Comment:
we need sync java & C++ so prepareResult should be inside searchRouteInternalPrepare

Reply:
this is just **hotfix** for current master. makePrecise/prepareResult should be fixed everywhere, in separate request(s)

Reply:
Hotfix is in place. searchRouteInternalPrepare should run prepare inside function
- Based on function name
- Cause Java has it inside